src/domain/session/room/timeline/deserialize.js

```diff
@@ -389,7 +398,7 @@ export function tests() {
     };
 
     function test(assert, input, output) {
-        assert.deepEqual(parseHTMLBody(platform, null, input), new MessageBody(input, output));
+        assert.deepEqual(parseHTMLBody(platform, null, true, input), new MessageBody(input, output));
```

Comment:
can we perhaps have a test for filtering out the nested tags of mx-reply when this flag is false?